src/main/java/org/pac4j/vertx/handler/impl/RequiresAuthenticationHandler.java

```diff
@@ -57,15 +39,16 @@
     private static final Logger LOG = LoggerFactory.getLogger(RequiresAuthenticationHandler.class);
 
     protected final Config config;
-    protected final String clientName;
+
+    protected final String clientNames;
```

Comment:
I would like to push for xxxNames rather than xxxs, to make it clear what the values actually represents (lists of names, not lists of the actual clients/authorizers/matcher objects)